project-koku/koku
# COST-881 Use contains instead of in for arrays

## Summary
This should allow us to do direct comparisons for access control. The previously used `in` operator was being used for single items which was forcing a string into a list of characters. 

## Testing
You can now view OCP on AWS/Azure/All API endpoints when rbac access is restricted on project.

1. Add `"access": {"openshift.project": {"read": ["analytics"]}}},` to the `DEVELOPMENT_IDENTITY` in your .env or `DEFAULT_IDENTITY` in settings.py
2. Run make load-test-customer-data
3. Check that API does not fail.
4. Remove access 
5. See API fail
